routes/DonationRoute.js

```diff
@@ -2,15 +2,20 @@ const router = require("express").Router();
 const {
   validateFlutterwaveWebhookRequest
 } = require("../middleware/verifyFlutterwaveWebhookRequest");
+const {
+  canRecordBulkDonations
+} = require("../middleware/canRecordBulkDonation");
 const {
   makeDonation,
+  recordBulkDonation,
   handleFluttwerwaveRequests,
   getTotalDonationsWithinTimeRange,
   getDonations,
   getTotalDonorsAndAmount
 } = require("../controller/DonationController");
 
 router.post("/", makeDonation);
+router.post("/bulk-sum", canRecordBulkDonations, recordBulkDonation);
```

Comment:
Hey @Chiemezuo, let's rename this route to `/manual` instead of `/bulk-sum` because that's what this route is for really. We use this to manually add donations.